### ISSUE TITLE
[BUG] Accessibility - Web App - Missing main landmark

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5139]
**Describe the bug**
Each page in a website must have at least one main landmark, used to identify Main content. AInstpector states “A main landmark provides a navigation point to the primary content of the page for users of assistive technologies. Most pages only need one main landmark, but in the case of portals or mashups, there may be more than one main landmark on a "page".” This web app version is missing these landmarks. 

[WGAC 2.4.1 Bypass Blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks)

**To Reproduce**
1. Go to app.illinois.edu
2. Click Continue through Orientation screens until Privacy screen

Run AInspector extension on Firefox, it identified this issue. 

**Expected behavior**
Each page in a website must have at least one main landmark

**Actual behavior**
These pages are all missing landmarks. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/ad3c1c7b-40e3-44ee-863e-063c551d6941)

![Image](https://github.com/user-attachments/assets/f487a015-062a-4fa3-b177-e336248ff9dd)

**Desktop (please complete the following information):**
PC:
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**
